internal/app/server/server.go

```diff
@@ -44,6 +52,18 @@ func RunWithContext(ctx context.Context, cancel context.CancelFunc, bootstrapCon
 		logger = log.New(bootstrapConfig.Logging.Level.String())
 	}
 
+	// Configure admin server.
+	adminServer := &http.Server{
+		//TODO(lisalu): Make below address configurable.
```

Comment:
Do you plan to make this configurable as a part of this PR?

Reply:
Yep